test/src/unittests/rhythm/test_rhythmextractor2013.py

```diff
@@ -64,167 +66,108 @@ def _assertEqualResults(self, result, expected):
-        result = self._runInstance(input, method="degara" )
+        expected = [0.0, [], 0.0, [], []] 
+        result = self._runInstance(input, method="degara")
         self._assertEqualResults(result, expected)
 
     def _createPulseTrain140(self): 
-        return(self._pulseTrain(bpm=140, sr=44100., offset=.1, dur=10))
+        return self._pulseTrain(bpm=140, sr=44100., offset=.1, dur=10) 
 
     def _createPulseTrainCombo(self):
-        # Define impulse train at 90bpm no offset
         impulseTrain90 = self._pulseTrain(bpm=90., sr=44100., offset=0., dur=20.)
-        # Define impulse train at 140bpm no offset
         impulseTrain140 = self._pulseTrain(bpm=140., sr=44100., offset=.1, dur=10.)
-        # Define impulse train at 200bpm with offset
         impulseTrain200 = self._pulseTrain(bpm=200., sr=44100., offset=.2, dur=10.)
-        # Define impulse train at 200bpm with no offset
         impulseTrain200b = self._pulseTrain(bpm=200., sr=44100., offset=0., dur=25.)
-        # Define Combination impulse train at 140-90-200-200 bpm
-        return(impulseTrain90 + impulseTrain140 + impulseTrain200 + impulseTrain200b)
+        # Result is a combined impulse train with 90-140-200-bpm and different offsets.
+        return impulseTrain90 + impulseTrain140 + impulseTrain200 + impulseTrain200b 
     
-    def testImpulseTrainMultifeature(self):
-        # Define expected values
+    def testImpulseTrain140Multifeature(self):
         impulseTrain140=self._createPulseTrain140()
+        bpm, ticks, confidence, estimates, bpmIntervals = self._runInstance(impulseTrain140,method="multifeature")
+        
         expectedBpm = 140
         expectedTicks = [i/44100. for i in range(len(impulseTrain140)) if impulseTrain140[i]!= 0]
-        expectedConfidence =3.8
-        #print("\nExpected Ticks Multifeature\n")
-        #print(expectedTicks)
-        # Test Multifeature with impulseTrain140
-        result = self._runInstance(impulseTrain140,method="multifeature")
-        # Check bpm
-        self.assertAlmostEqual(result[0], expectedBpm, 1e-2)
-        # Check ticks
-        self._assertVectorWithinVector(result[1], expectedTicks, 0.2)
-        #print("\nActual Ticks Multifeature\n")
-        #print(result[1])
-        # Check confidence
-        self.assertAlmostEqual(result[2], expectedConfidence,1.0)
-        # Check estimated bpm
-        for i in range(len(result[3])):
-            self.assertAlmostEqual(result[3][i], expectedBpm, 1.)
-        # Check bpm intervals
-        for i in range(len(result[4])):
-            self.assertAlmostEqual(result[4][i], 60./expectedBpm, 0.2)
-        # Test Multifeature with this combo impulseTrain
-        result = self._runInstance(self._createPulseTrainCombo(),method="multifeature")
-        # Define expected values
-        impulseTrainCombo= self._createPulseTrainCombo()
+        expectedConfidence =3.8 # This value was found by doing sample run tests.
```

Comment:
I maybe good to add a comment that current tolerance values were adjusted based on current output of the algorithm, to make the test pass, but that they make sense (e.g., 0.2 BPM tolerance  is very decent, 200 ms tolerance for ticks is an ok compromise, not sure what to comment on the tolerance for confidence, etc.)